client/jetpack-onboarding/steps/summary-completed-steps.jsx

```diff
@@ -0,0 +1,49 @@
+/** @format */
```

Comment:
Just noting that I'd also prefer if this file lived in the root directory of JPO (with regards to the PR description).

Reply:
See PR desc 😛 

> In a subsequent step, we will move both this file and `summary-todo-steps` one folder up.

It's easier to do it in this order: Once both components live in separate files, I can move them one level up _and_ change their `className`s at the same time.